src/lib/discord/bot.ts

```diff
@@ -13,13 +17,261 @@ new Command({
 		required: true,
 	} as const)
 	.handle(async i => {
+		const name: string = i.options.name
+		const entry: Entry | null = await fetchEntry(createSlug(name))
+
+		if (!entry) {
+			return {
+				content: `Requested entry "${name}" does not exist, or is unavailable.`,
+				color: 0x650ac9,
+			}
+		}
+
+		let content: string
+
+		if (entry.content.length > 985) {
+			content = `${entry.content.slice(0, 985)}...`
+		} else {
+			content = entry.content
+		}
 		return {
 			embeds: [
 				{
-					title: `Entry "${i.options.name}"`,
+					title: name,
+					url: `${BASE_URL}/entry/${entry.slug}`,
+					description: content,
+					color: 0x650ac9,
 				},
 			],
 		}
 	})
 
+new Command({
+	name: 'random',
+	description: 'View a random Repldex entry',
+}).handle(async data => {
+	const entryCount: number = await countEntries()
+	const entries = await fetchEntries({
+		limit: 1,
+		skip: Math.floor(Math.random() * entryCount),
+	})
+	const entry = entries[Math.floor(Math.random() * entries.length)]
+
+	return {
+		embeds: [
+			{
+				title: entry.title,
+				url: `${process.env.BASE_URL}/entry/${entry.slug}`,
+				description: entry.content,
+				color: 0x650ac9,
+			},
+		],
+	}
+})
+
+// This command has like 2 million issues, so i have temporaily removed it!
```

Comment:
I don't think this comment was meant to stay here lol

Reply:
Yeah, it wasn't suppose to stay lmao